src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsEvolveObject.java

```diff
@@ -39,6 +43,12 @@ public void verifyEvolveCall() {
     verifyNoEvolveInSlot(AdsContent.PREFOOTER_RIGHT);
   }
 
+  public void verifyEvolveCallMercury() {
```

Comment:
You are right - it should be consistent :+1: 

*Note: I don't like the approach to keeping 2 methods related to different platforms (skins) in one class however separating them would require some changes in whole tests architecture.*